.github/workflows/workflow-manual-run.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             - name: Test KFP worflow run
               timeout-minutes: 120
               run: |
-                  source kind/requirements.env
+                  source scripts/k8s-setup/requirements.env
```

Comment:
here we mention `scripts/k8s-setup` only twice, but might be better to have a variable anyway

Reply:
Fixed. Thanks